library/board.py

```diff
@@ -147,12 +159,81 @@ def redraw_gridlines(
             else:
                 print(term.move_x(x) + crosses)
 
+    def victory_subgrid(
+        self,
+        term: blessed.Terminal,
+        subgrid: npt.NDArray[np.str_],
+        start_coords: typing.Tuple[int, int],
+        winner: str,
```

Comment:
stringly typed things aren't great: try a `Optional[Literal['X', 'O']]`?

See: https://wiki.c2.com/?StringlyTyped

Reply:
Changed to this typing:
`typing.Optional[typing.Literal["X", "O", "None"]]`

Reply:
If you still have a `"None"` then it's useless to have a `typing.Optional` around, was suggesting you replace the `None` string with a `None` value.

Reply:
I have replaced "None" with None and changed `typing.Optional[typing.Literal["X", "O", "None"]]` to `typing.Optional[typing.Literal["X", "O"]]`